app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js

```diff
@@ -0,0 +1,631 @@
+        // Date.
+        docFrag = document.createDocumentFragment();
+        spanNode = docFrag.appendChild(document.createElement('time'));
+        spanNode.setAttribute('datetime', createdOn.toISOString());
+        spanNode.appendChild(
+            document.createTextNode(createdOn.toCustomISODateTime()));
+        html.replaceContent(document.getElementById('dd-date'), docFrag);
+
+        // Status.
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-status'), html.nonavail());
+
+        // Errors
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-errors'), html.nonavail());
+
+        // Warnings
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-warnings'), html.nonavail());
+
+        // Arch.
+        html.replaceContent(
+            document.getElementById('dd-suite-arch'),
+            document.createTextNode(arch));
+
+        // Time.
+        html.replaceContent(
+            document.getElementById('dd-suite-test-time'),
+            document.createTextNode(testTime.toCustomTime()));
+    }
+
+    function getBootLog(response) {
+        var bootLog;
+        var deferred;
+        var pathURI;
+        var result;
+        var serverURI;
+        var serverURL;
+        var translatedURI;
+        var lab;
+        // We only have one result.
+        result = response.result[0];
+        bootLog = null;
+
+        function bootLogDone(bootLog, bootResponse) {
+            var bootResult;
+
+            bootResult = bootResponse.result[0];
+            serverURL = result.file_server_url;
+            lab = result.lab_name;
+
+            if (!serverURL) {
+                serverURL = gFileServer;
+            }
+
+            translatedURI = u.createFileServerURL(serverURL, bootResult);
+            serverURI = translatedURI[0];
+            pathURI = translatedURI[1];
+
+            bootLog = tboot.createBootLog(
+                bootResult.boot_log,
+                bootResult.boot_log_html,
+                lab,
+                serverURI,
+                pathURI
+            );
+            gLogHref = u.getHref(serverURI, [
+                        pathURI,
+                        lab,
+                        bootResult.boot_log_html
+                    ]);
+            if (bootLog) {
+                html.replaceContent(
+                        document.getElementById('dd-suite-test-log'), bootLog);
+            } else {
+                html.replaceContent(
+                    document.getElementById('dd-suite-test-log'), html.nonavail());
+            }
+        }
+
+        // Boot & Test Log
+        if (result.boot_id) {
+            deferred = request.get('/_ajax/boot', {id: result.boot_id.$oid});
+            $.when(deferred)
+                .done(function(bootResponse) {
+                    bootLogDone(bootLog, bootResponse);
```

Comment:
Removing the `bootLog` argument from the `bootLogDone` function, this `.done` call can be simplified into:
```
.done(bootLogDone);
```